### PR TITLE
Added read documentation links for foreign key on ToolJet database dashboard drawers

### PR DESCRIPTION
**Pull request by S-Abdul-Rahman**
Resolves : 

#9696

### CODE REVIEW

File: frontend/src/_ui/Drawer/DrawerFooter/index.jsx [hunk 1]
```diff
@@ -79,20 +80,45 @@ function DrawerFooter({
     editForeignKeyInCreateTable,
   ]);
 
+  const isReadDocsTrue =
```
Comment (akshaysasidrn) [inline frontend/src/_ui/Drawer/DrawerFooter/index.jsx:83]: `isDrawerWithDocumentation` makes sense?

File: frontend/src/_ui/Drawer/DrawerFooter/index.jsx [hunk 2]
```diff
@@ -79,20 +80,45 @@ function DrawerFooter({
...
-            <a className="read-documentation">Read documentation</a>
-          </div>
-        </ToolTip> */}
+      <div
+        className={cx(
+          { 'd-flex justify-content-end drawer-footer-btn-wrap': !isReadDocsTrue },
+          { 'd-flex justify-content-between drawer-footer-btn-wrap': isReadDocsTrue }
+        )}
+      >
+        {(initiator === 'CreateTableForm' ||
```
Comment (akshaysasidrn) [inline frontend/src/_ui/Drawer/DrawerFooter/index.jsx:98]: can reuse conditional variable here?

**Review by akshaysasidrn — CHANGES_REQUESTED** [review]

File: frontend/src/_ui/Drawer/DrawerFooter/index.jsx (hunk 1 above)
Comment (S-Abdul-Rahman) [reply frontend/src/_ui/Drawer/DrawerFooter/index.jsx:83]: yes, makes sense 👍

**Review by akshaysasidrn — APPROVED** [review]

File: frontend/src/_ui/Drawer/DrawerFooter/index.jsx [hunk 3]
```diff
@@ -97,15 +92,15 @@ function DrawerFooter({
...
+            message={'Foreign key relations checks for referential integrity between two tables. Read more.'}
             placement="top"
             tooltipClassName="tootip-table read-docs-fk"
             show={showToolTipForFkOnReadDocsSection}
           >
             <div className="d-flex align-items-center">
               <Student />
               <a
-                href="https://docs.tooljet.com/docs/tooljet-db/database-editor/#foreign-key"
+                href="https://docs.tooljet.com/docs/tooljet-db/database-editor"
```
Comment (akshaysasidrn) [inline frontend/src/_ui/Drawer/DrawerFooter/index.jsx:103]: We don't need this change, needs to point to the foreign key

File: frontend/src/_ui/Drawer/DrawerFooter/index.jsx [hunk 4]
```diff
@@ -80,12 +80,7 @@ function DrawerFooter({
     editForeignKeyInCreateTable,
   ]);
 
-  const isDrawerWithDocumentation =
-    initiator === 'CreateTableForm' ||
-    initiator === 'EditTableForm' ||
-    initiator === 'CreateColumnForm' ||
-    initiator === 'EditColumnForm' ||
-    initiator === 'ForeignKeyTableForm';
+  const isDrawerWithDocumentation = initiator === 'ForeignKeyTableForm';
```
Comment (akshaysasidrn) [inline frontend/src/_ui/Drawer/DrawerFooter/index.jsx:83]: Don't we need to have links for other forms too to their corresponding doc as per design?

**Review by akshaysasidrn — CHANGES_REQUESTED** [review]

File: frontend/src/_ui/Drawer/DrawerFooter/index.jsx [hunk 5]
```diff
@@ -100,7 +105,17 @@ function DrawerFooter({
             <div className="d-flex align-items-center">
               <Student />
               <a
-                href="https://docs.tooljet.com/docs/tooljet-db/database-editor"
+                href={
+                  initiator === 'ForeignKeyTableForm'
```
Comment (akshaysasidrn) [inline frontend/src/_ui/Drawer/DrawerFooter/index.jsx:109]: nit: rather move this conditional as data?

tip:
```js
const drawerDocumentationsLinks = {
  CreateTableForm: 'link',
  EditTableForm: 'link',
   .....
}
const drawerNames = Object.keys(drawerDocumentationsLinks)
const isDrawerWithDocumentation = 
  drawerNames.some((drawerName) => drawerName === initiator)
```

**Review by akshaysasidrn — APPROVED** [review]

**Pull request merged**